ci/sparklingWaterPipeline.groovy

```diff
@@ -76,7 +76,9 @@ def withSharedSetup(sparkMajorVersion, config, code) {
                 checkout scm
                 config.commons = load 'ci/commons.groovy'
                 config.put("sparkVersion", getSparkVersion(config))
-
+                def kubernetesBoundaryVersionLine = readFile("gradle.properties").split("\n").find() { line -> line.startsWith('kubernetesSupportSinceSpark') }
```

Comment:
Haven't you already implemented reading such properties via a conversion to a map?

Reply:
I have implemented it when I wrote ci/Jenkinsfile-kubernetes from scratch. There is still JIRA for doing that for release in tests pipeline. I prefer to do that separately